wizard/state.py

```diff
@@ -53,3 +62,75 @@ class Metadata:
 
     def __bool__(self):
         return self.name != "" and self.url != ""
+    
+@dataclasses.dataclass
+class Croissant:
+    metadata: Metadata = Metadata()
+    distributions: list[Distribution] = dataclasses.field(default_factory=list)
```

Comment:
`distributions`/`record_sets` are under Metadata in mlcroissant. Should that also be the case here in the editor? I.e. should we follow the same structure to limit the overhead of learning two representations?

Note: in Croissant, distribution is singular because of https://schema.org/distribution. Files or resources could be alternative names that do not alter the original meaning of `distribution`.  Like this you can also write later:

```python
for resource in croissant.resources:

# or

for file in croissant.files:

# or

for file in croissant.distribution:
```

Reply:
At some point I'd like to talk to the people designing this stuff, as they and I have some disagreements on how structured data should be shaped, done though.

Reply:
Interesting. What shape/structure would you rather have?